packages/frontend/src/scripts/charts/renderer/getPointGroups.test.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'earl'
+import { Point } from './ChartRenderer'
+import { getPointGroups } from './getPointGroups'
+
+describe(getPointGroups.name, () => {
```

Comment:
You could make the assumption that dashed is only at the end and simplify the code.
Also each line can be dashed/not separately.
Also, how does this work for stacked chart?